doc/source/tutorial-series-get-started-with-flower-pytorch.ipynb

```diff
@@ -640,9 +620,21 @@
    "toc_visible": true
   },
   "kernelspec": {
-   "display_name": "flower-3.7.12",
+   "display_name": "flwr-rtwXnbAq-py3.10",
    "language": "python",
    "name": "python3"
+  },
+  "language_info": {
+   "codemirror_mode": {
+    "name": "ipython",
+    "version": 3
+   },
+   "file_extension": ".py",
+   "mimetype": "text/x-python",
+   "name": "python",
+   "nbconvert_exporter": "python",
+   "pygments_lexer": "ipython3",
+   "version": "3.10.6"
```

Comment:
@adam-narozniak do we need this change?

Reply:
No, we don't need it. Our current `nbstripout` removed it. That was added in Javier's run and I didn't notice it.